src/batteries.js

```diff
@@ -1,5 +1,32 @@
+import { causing } from "./cause";
 import { effector } from "./effect";
 import { trigger } from "./event";
+import { db } from "./db";
+
+/**
+ * Cause that returs the state of the database signal.
+ *
+ * @see {@link db}
+ */
+causing("db", () => db.value());
+
+/**
+ * Cause that returns the current time.
+ */
+causing("now", () => Date.now());
```

Comment:
I really like the terse look-and-feel of these lowercase causes - but this just came to mind: In SQL there a `NOW` function.

Are we making it hard to distinguish between built-in causes and user-created ones?

I'm not opposed to this change at all, just wanted to pose the question for us to think about.

What do you think?